Makefile

```diff
@@ -203,19 +203,19 @@ $(1)-lint: $(GEN_RS) $(GEN_TS)
 	$$(cargo) clippy  -p $(1) --all-features -- -D clippy::all -D warnings
 
 .PHONY: $(1)-test
-$(1)-test: $(GEN_RS) $(GEN_TS) auraed
+$(1)-test: $(GEN_RS) $(GEN_TS) $(1)-lint
 	$(cargo) test  -p $(1) --locked
 
 .PHONY: $(1)-test-all
-$(1)-test-all: $(GEN_RS) $(GEN_TS) auraed
+$(1)-test-all: $(GEN_RS) $(GEN_TS) $(1)-lint
```

Comment:
line 91 above defines `test` which also lints alongside tests.  if we don't need that any more then simplify the above `test` to only call `xxx-test` and then rely on this dep to lint first.